src/ceph-disk/ceph_disk/main.py

```diff
@@ -1279,11 +1279,12 @@ def get_dmcrypt_key(
         osd_uuid = get_oneliner(path, 'osd-uuid')
         ceph_fsid = read_one_line(path, 'ceph_fsid')
         if ceph_fsid is None:
-            raise Error('No cluster uuid assigned.')
-        cluster = find_cluster_by_uuid(ceph_fsid)
-        if cluster is None:
-            raise Error('No cluster conf found in ' + SYSCONFDIR +
-                        ' with fsid %s' % ceph_fsid)
+            cluster = 'ceph'
```

Comment:
A `LOG.warning("no `ceph_fsid` found at %s, falling back to 'ceph' for cluster name", ceph_fsid)` would be useful here to know when this is happening